meson.build

```diff
@@ -7,7 +7,11 @@ project('nwg-launchers', 'cpp',
 	version: '0.2.0'
 )
 
-add_global_link_arguments('-lstdc++fs', language: 'cpp')
+if host_machine.system().startswith('freebsd')
```

Comment:
Instead of hardcoding a list of OSes with libc++ by default only to miss some maybe check if the library is available e.g.,
 https://github.com/Alexays/Waybar/blob/e3fdd6d94a3e/meson.build#L22-L28

Reply:
Thanks I'll do that